go/gorilla/security/audit/handler-assignment-from-multiple-sources.yaml

```diff
@@ -1,5 +1,9 @@
 rules:
   - id: handler-assignment-from-multiple-sources
+    metadata:
+      cwe: "CWE-289: Authentication Bypass by Alternate Name"
+      owasp: "A2: Broken Authentication"
+      owaspapi: "API1: Broken Object Level Authorization"
```

Comment:
nit: thoughts on `owasp-api` vs. `owaspapi` :D

Reply:
Thoughts are: Let's get this PR in and if it's important later, we can make a ticket to change

Reply:
Ack. Once we start to use these fields (which will be immediately) it will be a pain to change. Agree with the general philosophy though.